<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnWindowsClass</class>
 <widget class="QMainWindow" name="LearnWindowsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LearnWindows</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1" colspan="2">
        <widget class="QPushButton" name="saveWindowsButton">
         <property name="text">
          <string>Save windows</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="nextImageButton">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QWidget" name="workArea" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="prevImageButton">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QPushButton" name="exportSamplesButton">
         <property name="text">
          <string>Export samples</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="undoNonRegion">
         <property name="text">
          <string>Undo Neg</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="undoWindowRegion">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QPushButton" name="clearRegionsButton">
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="toggleNon">
         <property name="text">
          <string>Hide Neg</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="exportTitle">
         <property name="text">
          <string>sample</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="sampleBackgroundsButton">
         <property name="text">
          <string>Sample Negs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>192</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFlie">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
   </widget>
   <addaction name="menuFlie"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="learnwindows.qrc"/>
 </resources>
 <connections/>
</ui>
